src/app/core/services/about-us.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { AboutUs } from '../interfaces/about-us';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AboutUsService {
+
+  private _url: string = "assets/mockinfo/mock-about-us.json"
```

Comment:
Be consistent, use only single quotes.
private _url: string = 'assets/mockinfo/mock-about-us.json';